Tools/MediaDriverTools/KernelBinToSource/KernelBinToSource.cpp

```diff
@@ -445,10 +569,15 @@ void printUsage(const std::string &sProgram)
               << " (" << PARAM_I << " InPath)"
               << " [" << PARAM_O << " OutPath]"
               << " [" << PARAM_V << " VarName]"
+              << " [" << PARAM_INDEX << " Kernel Index]"
+              << " [" << PARAM_TOTAL << " Kernel Total]"
               << std::endl
               << "    " << PARAM_I << " Path to Kernel binary input file (required)"             << std::endl
               << "    " << PARAM_O << " Path to Kernel binary output directory (optional)"       << std::endl
-              << "    " << PARAM_V << " Variable Name on the generated source file (optional)"   << std::endl;
+              << "    " << PARAM_V << " Variable Name on the generated source file (optional)"   << std::endl
+              << "    " << PARAM_INDEX << " Variab kernel Index  (optional)"                     << std::endl
+              << "    " << PARAM_TOTAL << " Variable kernel total count (optional)"              << std::endl;
```

Comment:
I don't understand what these variables do. Especially I don't understand what Index is for (total is I guess just total kernels count, but Index is a mystery to me).

Reply:
Please, respond.

Reply:
Index is for kernel offset. this situation we use 14 as kernel index

Reply:
I still don't understand what is index needed for. Why it is 14? Please, explain.